plugins/hls-refactor-plugin/test/Main.hs

```diff
@@ -1996,7 +1995,7 @@ suggestImportDisambiguationTests = testGroup "suggest import disambiguation acti
     compareHideFunctionTo = compareTwo "HideFunction.hs"
     withTarget file locs k = runWithExtraFiles "hiding" $ \dir -> do
         doc <- openDoc file "haskell"
-        void $ expectDiagnostics [(file, [(DiagnosticSeverity_Error, loc, "Ambiguous occurrence") | loc <- locs])]
+        void $ expectDiagnostics [(file, [(DiagnosticSeverity_Error, loc, "Ambiguous occurrence", Nothing) | loc <- locs])] -- TODO: Give this a proper error
```

Comment:
Are we going to finish this TODO in this PR?

Reply:
I think it should be `GHC-87543`

Reply:
There are several others with `Nothing` in the place of the code. I don't know why this todo was only put on this particular occurrence. There are probably some errors that don't have codes, but I think many of the current `Nothing`s could be replaced. 

I guess I'll try replacing this one and put a note in the list of future work that we should check the rest.

Reply:
Turns out GHC 9.6 doesn't have an error code for this, but GHC 9.8 does have it: `GHC-87110`

Reply:
I don't really want to open the can of worms of changing the tests based on which GHC version is used, so I think leaving it as is might be the best option. Do you agree?

Reply:
Yes, probably this is the best, we can wait until the version updated and then finish the todo by then